parser.yy

```diff
@@ -270,7 +270,7 @@ Expression:
 |	Expression '-' Expression
 |	Expression '.' Expression
 |	Expression '/' Expression
-|	Expression ':' Expression
+|	PATTERN ':' Term
```

Comment:
Good point. This change would not allow for:

a + _a : k + b // FullForm

Which is Plus[a,Optional[Blank[a],Plus[b,k]]]. I'll need to take another look at this.